packages/usa-table/src/usa-table--sortable/usa-table--sortable.twig

```diff
@@ -1,5 +1,7 @@
+{% if scrollable %}
 <div class="usa-table-container--scrollable" tabindex="0">
-  <table class="usa-table">
+{% endif %}
+  <table class="usa-table {{sticky_header ? 'usa-table--sticky-header' : ''}}">
```

Comment:
It'd be easier to update the class name if you define it once and re-use later.

```twig
{% if sticky_header %}
  {% set sticky_header_class = "usa-table--sticky-header" %}
{% endif %}

  <!-- Allows you do do something like this. -->
  <table class="usa-table {{ sticky_header ? sticky_header_class }}">
```

Reply:
Updated in [cae00e5](https://github.com/uswds/uswds/pull/5420/commits/cae00e59eec298dd295a202d63d3cc280a56efea)